instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/UpcomingRouteObjectsTest.kt

```diff
@@ -82,6 +108,334 @@ class UpcomingRouteObjectsTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::
         )
     }
 
+    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+    @Test
+    fun roadObjectsEuropeSanityTest() = sdkTest {
```

Comment:
@kried I added 2 tests to check the upcoming road objects integration. You can reuse some ideas of these tests for iOS (now they don't pass, because we're waiting for NN fixes).